src/web/mjs/connectors/BentoManga.mjs

```diff
@@ -0,0 +1,79 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class bentomanga extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'bentomanga';
+        super.label = 'Bentô Manga';
+        this.tags = [ 'manga', 'webtoon', 'novel', 'french' ];
+        this.url = 'https://bentomanga.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1'); 
+        let id = uri.pathname + uri.search;
+        let title = data[0].textContent.trim(); 
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(this.url + '/manga_list', this.requestOptions); 
+        let data = await this.fetchDOM(request, 'div.div-manga_cover a'); 
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.text.trim()
+            };
+        });
+    }    
+
+    async _getChapters(manga) {
+        let request = new Request(this.url + manga.id, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.component-chapter-title > a');
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, request.url),
+                title: element.querySelector('span.chapter_volume').innerText.trim() + ' ' + element.querySelector('span.chapter_title').innerText.trim(),
```

Comment:
Are there other elements than chapter_volume and chapter_title ? cant we just get element.text.trim() as title? 
Not a big deal tbh